### PR TITLE
#1 Add PyLaides code of conduct

### PR DESCRIPTION
**Pull request by dollyjung**
Connected to #1

### CODE REVIEW

File: src/code-of-conduct.html [hunk 1]
```diff
@@ -1,10 +1,17 @@
...
-            <img src="https://via.placeholder.com/524x343" alt='Logo' />
-        </div>
-        <div class="col">
-            <p>Text</p>
+    <div class="container">
+        <div class="row " style="padding-top: 50px;padding-bottom: 50px;">
+            <div class="col">
+                <img src="/static/assets/design/img-logo-python.png" alt='Logo' />                
+                <div style="position: absolute;top: 0px;">
+                    <p style="color: deeppink;font-style: bold;">PYLADIES BANGKOK</p>
```
Comment (dollyjung) [inline src/code-of-conduct.html]: font-style: bold doesn't work yet

File: src/code-of-conduct.html [hunk 2]
```diff
@@ -1,10 +1,17 @@
 <div class="container-fuild bg-white">
-    <div class="row ">
-        <div class="col">
-            <img src="https://via.placeholder.com/524x343" alt='Logo' />
-        </div>
-        <div class="col">
-            <p>Text</p>
+    <div class="container">
+        <div class="row " style="padding-top: 50px;padding-bottom: 50px;">
```
Comment (yothinix) [inline src/code-of-conduct.html]: Please move inline style to style.css, it's easier to make change later kub

**Comment by dollyjung** [comment]
> Please change all inline style to be store in `style.css` and using from there

already update inline style to style.css. Please kindly review :)

**Comment by protal** [comment]
Closes #1

**Pull request merged**